ignitionrobotics/ign-gazebo
# Allow user to modify joint type

# 🎉 New feature

## Summary

Adds support for editing the type of a joint in the  component inspector.

## Test it

1. `ign gazebo -v 4 pendulum_links.sdf`
2. Take note of the output from `ign model -m double_pendulum_with_base`
3. Select a joint in the entity tree.
4. Make some changes to the joint types in the component inspector. 
5. Rerun `ign model -m double_pendulum_with_base` and you should see the changes.

## Checklist

**Note to maintainers**: Remember to use **Squash-Merge**
